xclim/core/indicator.py

```diff
@@ -1684,6 +1686,7 @@ def _merge_attrs(dbase, dextra, attr, sep):
                 data, identifier=identifier, module=module_name
             )
 
+        # FIXME: Handle the potential exceptions explicitly
```

Comment:
Well I know this is what pylint whines about, but the point of this construct is that we actually don't know what could cause the function to fail. BUT, we don't want to stop the execution of the program if it does.
I don't think the construction makes any sense if we narrow it down to a list of exceptions.

Reply:
Understood, I don't think there's any issue with marking "exceptions" to the pylint rules, so long as we're explicit about them.